public/schemas/post-manifest.json

```diff
@@ -378,7 +378,7 @@
               "enum":["no_rejection","partial_rejection","full_rejection"]
             },
             "description": {
-              "description": "(14) Description of discrepancy",
+              "description": "(18) Description of discrepancy",
```

Comment:
this is line with what I am seeing on prod and the rest of the fields in this section. 90% sure it was a typo here.